<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PRMainWindow</class>
 <widget class="QMainWindow" name="PRMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PRMainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/pnrresources.qrc">
    <normaloff>:/new/icons/icons/packnroll128.png</normaloff>:/new/icons/icons/packnroll128.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>751</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>731</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textEdit">
         <property name="styleSheet">
          <string notr="true">QTextEdit {
	font-family: &quot;Monospace&quot;;
	font-size: 16pt;
}</string>
         </property>
         <property name="documentTitle">
          <string notr="true"/>
         </property>
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Source Sans Pro'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>4</y>
        <width>741</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>228</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_New"/>
    <addaction name="action_Save"/>
    <addaction name="action_SAVE_as"/>
    <addaction name="action_Update"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Options"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_Packnroll"/>
   </widget>
   <addaction name="menuF_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Options">
   <property name="text">
    <string>Op&amp;tions</string>
   </property>
  </action>
  <action name="action_Packnroll">
   <property name="text">
    <string>&amp;Pack n' Roll</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit Pack n Roll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_SAVE_as">
   <property name="text">
    <string>Sa&amp;ve as PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Update">
   <property name="text">
    <string>&amp;Update PKGBUILD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/pnrresources.qrc"/>
 </resources>
 <connections/>
</ui>
